INSTALL.md

```diff
@@ -133,4 +133,55 @@ target_link_libraries(foo PRIVATE ${OPENTELEMETRY_CPP_LIBRARIES})
 
 ## Build instructions using Bazel
 
-TBD
+### Prerequisites
+
+- A supported platform (e.g. Windows, macOS or Linux).
+Refer to [Platforms Supported](./README.md#supported-development-platforms)
+for more information.
+- A compatible C++ compiler supporting at least C++11.
+Major compilers are supported.
+Refer to [Supported Compilers](./README.md#supported-c-versions) for more information.
+- [Git](https://git-scm.com/) for fetching opentelemetry-cpp source code from repository.
+To install Git, consult the [Set up Git](https://help.github.com/articles/set-up-git/)
+guide on GitHub.
+- [Bazel](https://www.bazel.build/) for building opentelemetry-cpp API,
+SDK with their unittests. We use 3.7.2 in our build system.
+
+To install Bazel, consult the [Installing Bazel](https://docs.bazel.build/versions/3.7.0/install.html) guide.
+
+### Building as Standalone Bazel Project
```

Comment:
Isn't this more useful for contributors, less so installation?


E.g. I'd expect most bazel users to want  to do something like pulling in OTel CPP via `WORKSPACE` definition using `http_archive` or some other mechanism.

Not that this isn't useful, just not how I'd expect folks to consume.

Reply:
e.g. https://github.com/GoogleCloudPlatform/opentelemetry-operations-cpp/blob/main/WORKSPACE#L46

Or you can see:

https://github.com/googleapis/google-cloud-cpp/pull/1230/files for an example usage of OpenCensus C++

Reply:
This is a good point. We can have a separate section (say) "Incorporating Into An Existing Project", which includes the above-mentioned steps, similar to how we document for `CMake`.